### ISSUE TITLE
Not an issue, but..

### ISSUE THREAD

**Issue by LucasSaragosa** [samsonpianofingers/RTTIDumper#5]
Will this eventually support dumping the order of structs? for example in an executable: https://i.imgur.com/p5gOaB6.png
Theres quite a few, I would like to know the order they are in in the struct. And also a lot of the time defining a struct inside a struct as a variable without a pointer dumps the structs variables above it and makes it even more confusing (since thats also how its layed out in memory apart from alignment)
In the image, notice the spaces between the zeros? Maybe they mean something.

**Comment by GrandpaGameHacker** [comment]
That is implementation specific for every program.
Sometimes there is some metadata with custom RTTI implementations, but sadly accurate automatic structure recovery is a very very hard problem. MSVC does not include any hints by default.

Some versions I believe have the class size stored in the destructor code, but i've only seen this in a few games.

I am working on a structure dissector in my new ClassDumper2 gui tool similar to cheat engine, except it will include as much metadata as it can in how it tries to figure out what the structure is. I've got the barebones support in place, next I've got to code the gui parts.
